schemas/artifact-created-message.yml

```diff
@@ -41,8 +41,7 @@ properties:
           artifact.
         type:         string
         enum:
-          - s3
-          - azure
+          - blob
```

Comment:
We'd agreed that the message as it was being sent now is actively wrong and has no value and that we should change the message to mean actual completion instead of whatever the current event represents.

Once we've deprecated and blocked new azure/s3, we'll be stopping new artifacts being created with those schemes, so this will hopefully live for a very short amount of time.